self-core-impl/src/main/java/com/selfxdsd/core/StoredApiToken.java

```diff
@@ -0,0 +1,84 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.ApiToken;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * Stored API Token.
+ */
+public final class StoredApiToken implements ApiToken {
+    /**
+     * Token's name.
+     */
+    private final String name;
+
+    /**
+     * Secret.
+     */
+    private final byte[] secret;
```

Comment:
@andreoss same here, String.

Also, this class should also encapsulate the ``Storage`` (all Stored* entities follow this patter). However, there should be no accessor method for ``this.storage`` -- we will only use it internally, if needed.

Reply:
@amihaiemil Something is queried from `Storage` somehow?

Reply:
@andreoss actually, no. I was thinking to query the User owner, but it's better if we just inject it eagerly through the constructor.

So everytime we select an ApoToken we will also join the User owner.